dev/create-release/known_translations

```diff
@@ -416,3 +416,94 @@ zsxwing - Shixiong Zhu
 zuotingbing - Tingbing Zuo
 zuxqoj - Shekhar Bansal
 zzcclp - Zhichao Zhang
+Alfonso - Alfonso Buono
+Angerszhuuuu - Angerszhuuuu
+BOOTMGR - Harsh Panchal
+BelodengKlaus - Klaus Belodeng
+Byron - Byron
+Cedric-Magnan - Cedric-Magnan
+Daniel-Davies - Daniel Davies
+Darek - Darek Zoń
+Geek - Geek
+IonutBoicuAms - Ionut Boicu
+Jie - Jie
+Leona - Leona
+PengLei - Peng Lei
+Qian.Sun - Qian Sun
+RabbidHY - RabbidHY
+RoryQi - Rory Qi
+SaurabhChawla - Saurabh Chawla
+Shockang - Shockang
+TongWeii - Tong Weii
+Utkarsh - Utkarsh Srivastava
+Wang - Wang
+YangJie - Yang Jie
+Yikf - Yi Kaifei
+Yimin - Li Yimin
+alexander_holmes - Alexander Holmes
+beobest2 - Hyunwoo Park
+bjornjorgensen - Bjørn Jørgensen
+chenzhx - Zhixiong Chen
+copperybean - copperybean
+daugraph - daugraph
+dchvn - dchvn
+dgd-contributor - dgd-contributor
+dgd_contributor - dgd_contributor
+dohongdayi - dohongdayi
+erenavsarogullari - Eren Avsarogullari
+fhygh - fhygh
+flynn - Flynn
+gaoyajun02 - gaoyajun02
+gengjiaan - gengjiaan
+herman - Herman Geuvers
+hi-zir - hi-zir
+huangmaoyang2 - huangmaoyang2
+huaxingao - Huaxin Gao
+hujiahua - hujiahua
+jackierwzhang - jackierwzhang
+jackylee-ch - Jacky Lee
+jiaoqb - Qingbo Jiao
+jinhai - Jin Hai
+khalidmammadov - Khalid Mammadov
+kuwii - kuwii
+leesf - leesf
+mans2singh - mans2singh
+mcdull-zhang - mcdull-zhang
+michaelzhang-db - michaelzhang-db
+minyyy - minyyy
+nyingping - Nie Yingping
+pralabhkumar - Pralabh Kumar
+remykarem - Remy
+sandeepvinayak - Sandeep Pal
+senthilkumarb - Senthil Kumar Bhaskaran
+skhandrikagmail - skhandrikagmail
+sperlingxx - Alfred Xu
+sudoliyang - Liyang Zhang
+sweisdb - sweisdb
+sychen - Sung-Yu Chen
+tan.vu - Tan Vu
+tanel.kiis@gmail.com - tanel.kiis@gmail.com
+tenglei - tenglei
+tianhanhu - tianhanhu
+tianlzhang - tianlzhang
+timothy65535 - timothy65535
+tooptoop4 - tooptoop4
+vadim - Vadim Patsalo
+w00507315 - w00507315
+wangguangxin.cn - wangguangxin.cn
+wangshengjie3 - wangshengjie3
+wayneguow - Wei Guo
+wooplevip - wooplevip
+wuyi - Wu Yi
+xiepengjie - xiepengjie
+xuyu - xuyu
+yangjie01 - yangjie01
+yaohua - yaohua
+yi.wu - yi.wu
```

Comment:
This should be a map between GitHub ID and actual name IIRC.

@yi.wu seems invalid GitHub ID at least. We will have to remove these.